### PR TITLE
AA, Nemo, Hecke announcement

### PR DESCRIPTION
**Pull request by thofma**
First needs the Hecke release.

### CODE REVIEW

File: news/_posts/2020-12-23-ANH-release.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: AbstractAlgebra.jl 0.12.0, Nemo 0.19.0 and Hecke 0.9.0 released
```
Comment (rfourquet) [inline news/_posts/2020-12-23-ANH-release.md]: Maybe just "AbstractAlgebra" (without ".jl"), to be consistent with the other 2? (same below in the first sentence).

**Comment by rfourquet** [comment]
The only change that look breaking to me is 

> "Rename `rref` for matrices over rings to `rref_rational`."

for AbstractAlgebra. If you have in mind other breaking changes (as these are "breaking releases"), it could be useful to highlight them (e.g. simply by writing "(breaking)" next to them). Just a suggestion though, and a better place to track this might be in the respective manuals instead.

**Comment by thofma** [comment]
Yes, this is the only breaking change. I have added "(breaking)" to this item.

**Comment by rfourquet** [comment]
Thanks!

**Pull request merged**